src/js/Controllers/DisplayCapabilities.js

```diff
@@ -96,6 +96,7 @@ const mediaButtonCapabilities = [{
 	"name": "CUSTOM_BUTTON"
 }]
 
+
```

Comment:
Remove empty line

Reply:
@ValeriiMalkov that's as part of review answer

Reply:
He [asked](https://github.com/smartdevicelink/generic_hmi/pull/414#discussion_r686214581) to add empty line after `67` line of code, not after `98` that you've add